railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -21,6 +21,11 @@ source 'https://rubygems.org'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
+<%- if options.api? -%>
+# Use ActiveModelSerializers to serialize JSON responses
+gem 'active_model_serializers', '~> 0.10.0.rc1'
```

Comment:
> you don't want to implement every single serialization point.

I don't understand this. Can you give me some examples? You can use `json.render partial: ...` and I don't think you have to implement every single serialization point.